kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -166,6 +168,14 @@
   import LanguageSwitcherFooter from '../LanguageSwitcherFooter';
   import FacilityModal from './FacilityModal';
 
+  // https://davidwalsh.name/query-string-javascript
+  function getUrlParameter(name) {
+    name = name.replace(/[[]/, '[').replace(/[\]]/, '\\]');
+    var regex = new RegExp('[\\?&]' + name + '=([^&#]*)');
```

Comment:
Thought we had this somewhere already, but I think it was in Hashi and then I later removed it. Can also polyfill the URL constructor too for this.

Reply:
looks like [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#Browser_compatibility) is the new hotness

Reply:
Yeah, when you parse a url with the URL constructor, the it generates a URLSearchParams object for the query params.